utils/land-patterns.stanza

```diff
@@ -217,12 +217,19 @@ public defn soldermask-defined (lp:LandPattern):
 
   SMD-l
 
-public pcb-pad smd-pad (s:Shape):
+public pcb-pad smd-pad (copper-layer:Shape,solder-mask-layer:Shape|False,paste-layer:Shape):
```

Comment:
I like it! 

We also should update the docs, but that is outside this repo so out of scope for this PR.

Reply:
Created ticket to capture this